### PR TITLE
created directory with EOIR db schema and files

### PR DESCRIPTION
**Pull request by yaobviously**
Added the schema for the EOIR database under exploration by the data engineering team. 

## Type of change

New directory and several files that give an overview

## Checklist:

- [X] My code follows the style guidelines of this project
- [X]  I have performed a self-review of my own code
- [X] I have removed unnecessary comments/console logs from my code
- [X] I have made corresponding changes to the documentation if necessary (optional)
- [X] My changes generate no new warnings
- [X] I have checked my code and corrected any misspellings
- [X] No duplicate code left within changed files
- [X] Size of pull request kept to a minimum
- [X] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -0,0 +1,27 @@
+[[source]]
```
Comment (minh14496) [inline Pipfile]: Jason you shouldn't upload your Pipfile and Pipfile.lock to the github keep that local.

**Review by minh14496 — CHANGES_REQUESTED** [review]
Thanks Jason for doing fantastic job. You may need to add your EOIR data directory in .ebignore so that we don't accidentally deploy the app w the folder

File: Pipfile (hunk 1 above)
Comment (andrewlee977) [reply Pipfile]: I don't see an issue with the Pipfiles. Looks great otherwise, excited to see if this new data can improve the product!

**Review by andrewlee977 — APPROVED** [review]
Looks great! Thanks for taking this on!

File: Pipfile (hunk 1 above)
Comment (minh14496) [reply Pipfile]: There must be a reason why no Labs project repo nor even asylum FE BE and DS have Pipfile and Pipfile.lock. That's why I suggest don't push Pipfile to main.

**Review by Abdelapv53 — APPROVED** [review]
Looks great Jason!

File: Pipfile (hunk 1 above)
Comment (KanadeFuzen) [reply Pipfile]: I have seen problems being caused from team members uploading their Pipfile from previous projects; something that has to do with version control and OS. Although there should be no problem with Pipfile.lock or requirements.txt.

**Review by KanadeFuzen — CHANGES_REQUESTED** [review]
Everything looks good, Jason. Just to be safe, do not upload your Pipfile like Minh said, as that merge could interfere with someone's local repo when they git pull to update their repo to the newest version.

File: Pipfile (hunk 1 above)
Comment (yaobviously) [reply Pipfile]: Alright guys, my bad. I used git add . exactly as we were told not to do.

Won't happen again! Probably.

**Review by minh14496 — APPROVED** [review]
Terrific Job Jason. Thanks for deleting Pipfile

**Comment by mikecarrier4** [comment]
A template of all rows and PK and NNValues.  Looks good

**Review by mikecarrier4 — APPROVED** [review]
A document of all columns with NN and all rows in the EOIR DataSet.

**Review by KanadeFuzen — APPROVED** [review]
Looks good, sorry for not approving earlier!

**Pull request merged**